src/metabase/upload.clj

```diff
@@ -282,7 +282,7 @@
         (when (not (str/blank? value))
           (parser value))))))
 
-(mu/defn ^:private column-types-from-rows :- [:sequential (into [:enum] column-types)]
```

Comment:
why do you make it public? if it's for test purposes I suggest we should keep it private and use `#'` to access it.

Reply:
Yeah, I would like to keep it private but I couldn't commit unless I made this public because I got an error from the linter saying was it was undeclared in the `upload_test` namespace. No idea why it's coming up now